### PR TITLE
[ENH] add ax param to RejectLog.plot method

### LINKED ISSUE #241: Passing in figure/ax in reject_log plot
Right now, the figure size is hard-coded in reject log plot. It would be nice to allow one to instantiate it themselves and say pass in `ax`?

### PR DESCRIPTION
**Pull request by sappelhoff**
closes #241 

Thought this would be easy to support before the release.

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -58,11 +60,11 @@ Bug
...
 
 - Fixed bug in picking bad channels during interpolation. This bug only affects users who got an assertion
-  error when running :class:`autoreject.Autoreject`. Fixed by `Mainak Jas`_ in :github:`#115`
+  error when running :class:`autoreject.Autoreject`, by `Mainak Jas`_ in :github:`#115`
 - Added check for channel locations so that autoreject does not
-  hang when the channel positions are nan. Fixed by `Mainak Jas`_ in :github:`#130`
+  hang when the channel positions are nan, by `Mainak Jas`_ in :github:`#130`
 - Fixed bug in random seed for the Ransac algorithm when n_jobs > 1, by `Legrand Nico`_ and `Mainak Jas`_ in :github:`#138`
-- Fixed pikcling of :class:`autoreject.AutoReject`. Fixed by `Hubert Banville`_ in :github:`#193`
+- Fixed pikcling of :class:`autoreject.AutoReject`, by `Hubert Banville`_ in :github:`#193`
```
Comment (sappelhoff) [inline doc/whats_new.rst:67]:
```suggestion
- Fixed pickling of :class:`autoreject.AutoReject`, by `Hubert Banville`_ in :github:`#193`
```

File: autoreject/autoreject.py [hunk 2]
```diff
@@ -1257,7 +1262,10 @@ def plot(self, orientation='vertical', show_names='auto', show=True):
         if show_names == 'auto':
             show_names = 1 if len(self.ch_names) < 25 else 5
 
-        figure, ax = plt.subplots(figsize=(12, 6))
+        if not ax:
```
Comment (jasmainak) [inline autoreject/autoreject.py:1265]:
```suggestion
        if ax is not None:
```

**Comment by jasmainak** [comment]
Looks good other than nitpick

**Comment by jasmainak** [comment]
Thanks @sappelhoff !

**Pull request merged**